lib/list.gd

```diff
@@ -53,12 +53,34 @@ BIND_GLOBAL( "ListsFamily", NewFamily( "ListsFamily", IsList ) );
 ##
 #R  IsPlistRep  . . . . . . . . . . . . . . . . representation of plain lists
 ##
+##  <#GAPDoc Label="IsPlistRep">
 ##  <ManSection>
-##  <Filt Name="IsPlistRep" Arg='obj' Type='Representation'/>
+##  <Filt Name="IsPlistRep" Arg='obj' Type='representation'/>
 ##
 ##  <Description>
+##  &GAP; lists created by entering comma separated values in square brackets
+##  are represented internally as so-called <E>plain lists</E>,
```

Comment:
Note that this is not 100% correct: for boolean lists, GAP returns a blist rep:
```
gap> IsPlistRep( [ true, false ] );
false
gap> IsBlistRep( [ true, false ] );
true
```

Reply:
Good point. I will fix this.